lib/tasks/import_remediations.rake

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+def remove_ref_id_prefix(ref_id)
+  short_ref_id = ref_id.downcase.split('xccdf_org.ssgproject.content_profile_')[1];
+  short_ref_id.present? ? short_ref_id : ref_id
+end
+
+task import_remediations: :environment do
+  account = Account.find_by_account_number(ENV['JOBS_ACCOUNT_NUMBER'])
```

Comment:
We will need to set one of our accounts here, wanted to keep the number as a secret, so I kept this as an env var.